packages/sql-graphql/test/helper.js

```diff
@@ -116,4 +116,14 @@ module.exports.clear = async function (db, sql) {
     await db.query(sql`DROP TABLE generated_test`)
   } catch (err) {
   }
+
+  try {
+    await db.query(sql`DROP TABLE enum_tests`)
+  } catch (err) {
+  }
+
+  try {
+    await db.query(sql`DROP TYPE special_chars`)
+  } catch (err) {
+  }
```

Comment:
Could you please use one of the other table names that are already cleared?

Reply:
Sure, done on ef0fd99 👍🏼